### PR TITLE
removal: darkest theme references

### PR DESCRIPTION
**Pull request by Rajdeepc**
## Description

Removed `darkest` theme and its references from the SWC project. This is a first step towards deprecating the `lightest` & `darkest` themes and its files from the project. We will be continuing with only `light` and `dark` variants of `spectrum` and `express` and `spectrum-2` systems.

## Related issue(s)

-

## How has this been tested?

-   [ ] _Test case 1_
    1. Go here
    2. Do this
-   [ ] _Test case 2_
    1. Go here
    2. Do this

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [ ] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [ ] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -218,7 +218,7 @@ jobs:
                   keys:
                       - v2-golden-images-<< pipeline.parameters.current_golden_images_hash >>-hcm-
                       - v2-golden-images-main-hcm-
-                      - v2-golden-images-main-classic-lightest-medium-ltr-
+                      - v2-golden-images-main-classic-light-medium-ltr-
```
Comment (Westbrook) [inline .circleci/config.yml:221]: I think we can remove this all together. It was to give a backstop when we first added HCM support.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1028,7 +1028,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 -   **action-menu:** update spectrum css input ([62a5065](https://github.com/adobe/spectrum-web-components/commit/62a50651a3c797598f8d76330e87f2824cdabbc5))
 -   **actionbar:** add actionbar component package ([36dd7e9](https://github.com/adobe/spectrum-web-components/commit/36dd7e943a8b825b947f0053ddbe31b69ca27062))
 -   **actionbar:** update spectrum css input ([8c888aa](https://github.com/adobe/spectrum-web-components/commit/8c888aa883490fe8e6a070f799095284128a609a))
--   add "darkest" theme styles ([fe38025](https://github.com/adobe/spectrum-web-components/commit/fe38025d2b631f07834789c936efe7ffcbc1a628))
```
Comment (Westbrook) [inline CHANGELOG.md:1031]: This is a changelog, it shouldn't be edited.

File: packages/card/src/spectrum-card.css [hunk 3]
```diff
@@ -69,6 +69,10 @@ governing permissions and limitations under the License.
     --spectrum-card-selected-background-opacity: 0.1;
 }
 
+:host .spectrum--darkest {
```
Comment (Westbrook) [inline packages/card/src/spectrum-card.css:72]: This is wrong. Where does it come from?

File: projects/documentation/src/getting-started.ts [hunk 4]
```diff
@@ -10,5 +10,4 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import '@spectrum-web-components/theme/theme-lightest.js';
 import '@spectrum-web-components/theme/scale-medium.js';
```
Comment (Westbrook) [inline projects/documentation/src/getting-started.ts:13]: I'm not 100% sure how this is used anymore, but this should _likely_ be `large` and `dark` as seen in `getting-started.md` or removed all together. 🤔

File: test/visual/test.ts [hunk 5]
```diff
@@ -205,7 +205,7 @@ export const regressVisuals = async (name: string, stories: TestsType) => {
         if (color && scale && dir) {
             test(stories, name, color, scale, dir);
         } else {
-            const colors: Color[] = ['lightest', 'light', 'dark', 'darkest'];
+            const colors: Color[] = ['light', 'dark'];
```
Comment (Westbrook) [inline test/visual/test.ts:208]: In other places you _don't_ remove `darkest`?

File: tools/styles/CHANGELOG.md [hunk 6]
```diff
@@ -445,8 +445,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Features
 
--   add "darkest" theme styles ([fe38025](https://github.com/adobe/spectrum-web-components/commit/fe38025))
```
Comment (Westbrook) [inline tools/styles/CHANGELOG.md:448]: Please don't edit changelogs.

File: tools/theme/CHANGELOG.md [hunk 7]
```diff
@@ -433,8 +433,6 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Features
 
--   add "darkest" theme styles ([fe38025](https://github.com/adobe/spectrum-web-components/commit/fe38025))
```
Comment (Westbrook) [inline tools/theme/CHANGELOG.md:436]: Changelog.

**Review by Westbrook — COMMENTED** [review]
Please rebase this _behind_ @TarunAdobe's S2 token work.

**Comment by Rajdeepc** [comment]
This has been handled in the major S2 Foundation work.

**Pull request closed without merging**